torch/onnx/_internal/fx/export.py

```diff
@@ -144,41 +230,42 @@ def _wrap_fx_args_as_ts_args(g, root, node, fx_name_to_ts_value):
             else:
                 # Get default from schema.
                 complete_args.append(expected_arg.default_value)
+    # (2) retrive existing
     return tuple(
-        _retrieve_or_wrap_scalar_as_constant(
-            # The node.meta["val"] is generated by FakeTensorProp.
-            g,
+        _retrieve_or_adapt_input(
             arg,
-            fx_name_to_ts_value,
+            fx_name_to_onnxscipt_tensor,
+            # The node.meta["val"] is generated by FakeTensorProp.
             node.meta["val"],
         )
         for arg in complete_args
     )
 
 
-def _fill_tensor_types(ts_values, expected_values):
-    flat_ts_values, _ = _pytree.tree_flatten(ts_values)
-    flat_expected_values, _ = _pytree.tree_flatten(expected_values)
-    for ts_value, expected_value in zip(flat_ts_values, flat_expected_values):
-        ts_value.setType(torch._C.TensorType.create_from_tensor(expected_value))
+# def _fill_tensor_types(ts_values, expected_values):
+#     flat_ts_values, _ = _pytree.tree_flatten(ts_values)
+#     flat_expected_values, _ = _pytree.tree_flatten(expected_values)
+#     for ts_value, expected_value in zip(flat_ts_values, flat_expected_values):
+#         ts_value.setType(torch._C.TensorType.create_from_tensor(expected_value))
 
 
 def _export_fx_to_ts(fx_module_with_metadata, opset_version):
-    # TODO(wechi): To get rid of TorchScript dependency,
-    # "g" should just be onnx.GraphProto or an equivalent
-    # data structure in ONNXScript.
-    g = torch._C.Graph()
+
+    # Initialize the ONNX graph
+    onnxscript_graph = graph_building.TorchScriptGraph()
+    tracer = graph_building.TorchScriptTracingEvaluator(onnxscript_graph)
+
     # In the following loop, a TorchScript graph is created to
     # represent the input FX graph with ONNX symbols (e.g., onnx::add).
     # To connect the values to nodes in the TorchScript graph, we maintian
-    # fx_name_to_ts_value. Basically, we want to translate
+    # fx_name_to_onnxscipt_tensor. Basically, we want to translate
```

Comment:
Please do a file-wise replacement.
```suggestion
    # fx_name_to_onnxscipt_value. Basically, we want to translate
```

Reply:
Done